akka-cluster/src/main/scala/akka/cluster/protobuf/ClusterMessageSerializer.scala

```diff
@@ -372,7 +374,9 @@ final class ClusterMessageSerializer(val system: ExtendedActorSystem)
     MemberStatus.Exiting -> cm.MemberStatus.Exiting_VALUE,
     MemberStatus.Down -> cm.MemberStatus.Down_VALUE,
     MemberStatus.Removed -> cm.MemberStatus.Removed_VALUE,
-    MemberStatus.WeaklyUp -> cm.MemberStatus.WeaklyUp_VALUE)
+    MemberStatus.WeaklyUp -> cm.MemberStatus.WeaklyUp_VALUE,
+    MemberStatus.PreparingForShutdown -> cm.MemberStatus.PreparingForShutdown_VALUE,
+    MemberStatus.ReadyForShutdown -> cm.MemberStatus.ReadyForShutdown_VALUE)
```

Comment:
we need something for rolling update compatibility. and old node doesn't know about PreparingForShutdown_VALUE  and ReadyForShutdown_VALUE when deserializing

Reply:
Discussed offline. We won't fully support rolling updates and setting prepare for shutdown at the same time. Users will need to make sure a rolling upgrade to the first version that supports this feature is complete before using it. But we will do a release that ignores gossip that can't be deserialised first so it doesn't crash old nodes